test/known_issues/test-vm-getters.js

```diff
@@ -16,4 +16,9 @@ const context = vm.createContext(sandbox);
 const code = 'Object.getOwnPropertyDescriptor(this, "prop");';
 const result = vm.runInContext(code, context);
 
-assert.strictEqual(result, descriptor);
+// Ref: https://github.com/nodejs/node/issues/11803
+
+assert.deepStrictEqual(Object.keys(result), Object.keys(descriptor));
+for (const prop of Object.keys(result)) {
+  assert.strictEqual(result[prop], descriptor[prop]);
+}
```

Comment:
Yes, I did, but it did not work as such and I could not figure out exactly why. For the sake of time I put this as a 'working' version, hoping that somebody would suggest a more elegant way, if there is one.